File: day64/indal/indal.py
# class P:
#     a=10
#     def __init__(self):
#         self.name="Indal Kumar"
#         self.age=22
#     def m3(self):
#         print("instance method of P class")
#     @classmethod
#     def m1(cls):
#         print("Class Method of P")
    
#     @staticmethod
#     def m2():
#         print("Indal Kumar")
    
# class C(P):
#     def __init__(self):
#         super().__init__()
#         print("static varibale of parent class: -",super().a)
        
    
    # def display(self):
    #     super().m2()
class A:
    def m1(self):
        print("A method")
class B(A):
    def m1(self):
        print("B method")
class C(B):
    def m1(self):
        print("C method")
class D(A):
    def m1(self):
        super(B,self).m1()
d=D()
d.m1()

File: day66/indal/indal.js
// static method in javascript

class Indal {
  static mufun() {
    return -1;
  }
}
// console.log(Indal.mufun());

class Parent {
  constructor(name, age) {
    this.name = name;
    this.age = age;
  }
}
class Child extends Parent {
  constructor(name, age, marks) {
    super(name, age);
    this.marks = marks;
  }
}
let c1 = new Child("indal", 10, 20);
console.log(c1);
